src/activity_handlers.cpp

```diff
@@ -696,6 +708,18 @@ void activity_handlers::fill_liquid_do_turn( player_activity *act_, player *p )
             on_ground = source_stack.begin();
             std::advance( on_ground, act.values.at( 1 ) );
             liquid = *on_ground;
+			break;
+        case LST_MONSTER:
+            Creature *c = g->critter_at( source_pos );
+            source_mon = dynamic_cast<monster *>( c );
+            if( source_mon == nullptr ) {
+                debugmsg( "could not find source creature for liquid transfer" );
```

Comment:
I did a throw originally following  the example of the other cases, then Coolthulhu https://github.com/CleverRaven/Cataclysm-DDA/pull/21042#pullrequestreview-41364116 pointed out that it wasn't necessary. Should I change it to a throw?